ports/supermegazero/port.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 3,
+  "name": "supermegazero.zip",
+  "items": [
+    "Super Mega Zero.sh",
+    "supermegazero"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Super Mega Zero",
+    "porter": [
+      "Fraxinus88"
+    ],
+    "desc": "Miss Undefined has been whisked away by the dastardly Kill Screen. Go from Zero to Hero in this fast-paced platformer, chasing time, the high score, and eventually the Kill Screen itself.\n\nSuper Mega Zero is a tight precision and puzzle platformer with shoot 'em up elements. Simply touch an upgrade and switch forms to go from classic, straight-forward platforming to the more puzzle-like action number system or Shoot' Em Up gameplay.\n\nNote: Poor readability on small screens and options menu does not work.",
+    "desc_md": null,
+    "inst": "Purchase the game from: https://store.steampowered.com/app/1581880/Super_Mega_Zero/\nRun steam console via Windows+R: Type: steam://open/console. \nDownload the 20 April 2022 version by typing: download_depot 1581880 1581881 8562910234793311857 in the console, when the download is complete, it will tell you the location of the folder,  copy all the files from the download folder to /ports/supermegazero/gamedata",
```

Comment:
You have two instruction fields. One for the instructions you see on the device and one inst_md which will be shown on the website. 

It does not make sense to put the full steam link in the Portmaster install instructions on screen. Also you have no formating on the device.

So keep it short on the device instructions or tell them to visit the site for details and the detailed on the inst_md

Reply:
Updated. Kindly review :)